rollup/core/classes/RollupCalculator.cls

```diff
@@ -1011,13 +1011,16 @@ public without sharing abstract class RollupCalculator {
 
     public override void performRollup(List<SObject> calcItems, Map<Id, SObject> oldCalcItems) {
       SObjectType sObjectType = this.getCalcItemType();
+      // objIds is used as a bind variable in the resulting queryString
```

Comment:
Would it be better for the `RollupQueryBuilder` class to handle executing the query (instead of just generating the query string)? That way, you wouldn't have to create the mystery `objIds` variable here - `RollupQueryBuilder` is already using it internally

Reply:
I think I've shied away from doing exactly that because I called it _builder_ - but I could be convinced. I might not get to this change in this version, but I agree with the sentiment that it's time to consolidate the coupling (and I might do it this version, just want to see how many changes it will entail)

Reply:
I tooled around with these changes, and will definitely keep them in mind for a future release. There are a number of cascading changes between classes and tricky edge cases with existing query bind variables that made accomplishing this change larger than I would like at the moment.